tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -1306,4 +1306,126 @@ TestCarePartnerBrowsingWhileOnCall:
           Value: 'The call has ended'
   # Fill survey
     - Type: case
-      Value: PostCallSurveyCarePartner
\ No newline at end of file
+      Value: PostCallSurveyCarePartner
+
+# CXTB-124: Call Participant's device - Call Complete.
+TestCarePartnerCallParticipantsDevice:
```

Comment:
I would update this name because there are other cases that are Call Participant’s Device - Call Ignored, Call Participant’s Device - Call Missed.  The name TestCarePartnerCallParticipantsDevice is not detailed enough

Reply:
I updated the test name to be: TestCarePartnerCallParticipantsDeviceCallComplete